/uber

proger opened the pull request:
This PR upgrades java, gradle, groovy, grails to something recent and introduces a new endpoint, `/uber`:

```
$ curl -s  -X POST -H "Content-Type: application/json" -d "['Привіт, котанче. Як справи?', 'Ну шо, приїхали?']" http://localhost:8080/uber | jq -c '.[]'

[["Привіт, котанче. ",["Привіт","котанче"],["","привіт",","," ","котанче","."," "]],["Як справи?",["Як","справи"],["","як"," ","справа","?"]]]
[["Ну шо, приїхали?",["Ну","шо","приїхали"],["","ну"," ","шо",","," ","приїхати","?"]]]
```

The new endpoint accepts a batch of texts, processes all of them parallel and returns `[sentences, tokens, lemmas]` for each text.

Ping @dchaplinsky, let me know if you can use this.

proger committed:
upgrade gradle, grails, groovy to something recent and LT to 6.0

proger committed:
logback needs to be xml

proger committed:
lose swagger: could not get it to compile

proger committed:
/uber accepts batches of documents and returns [sentence, tokens, lemmas] for each

arysin commented:
Дуже дякую, але почекайте дещо. Я переписав проєкт наново з усіма новими платформами, за день-два закину в репозитарій.

arysin commented:
Вітання, я створив новий проєкт https://github.com/arysin/nlp_uk_api
Він повністю оновлений й має працювати, але я ще маю добити Dockerfile
Цей проєкт я архівую

proger committed:
Create docker-publish.yml

proger committed:
Run workflows on the uber branch

proger committed:
Update README.md

proger committed:
Update README.md

proger committed:
Update Dockerfile

proger committed:
Update Dockerfile

proger committed:
Update Dockerfile

proger committed:
Update README.md

proger committed:
Update README.md

proger committed:
UberService: use adjustTokens on words

code from https://github.com/brown-uk/nlp_uk/blob/master/src/main/groovy/ua/net/nlp/tools/TextUtils.groovy#L282-L303

proger committed:
some TokenizeOptions

proger committed:
make it compile

proger committed:
Dockerfile: no gradle daemon when building

proger committed:
match number of tokens for words and lemmas

proger committed:
update README example
